windows/gce-gpu-monitoring-cuda.ps1

```diff
@@ -0,0 +1,227 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https=//www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+ 
+<#
+This script can be used for monitoring  GPU utilization on Google Compute Engine
+instances with GPUs attached. It requires the nvidia-smi tool to be properly
+installed on the system (https=//developer.nvidia.com/nvidia-system-management-interface).
+
+This script assumes it is being executed on GCE instance and that it has access
+to the metadata server (https=//cloud.google.com/compute/docs/storing-retrieving-metadata).
+#>
+ 
+Param(
+    #configure the interval, default is 5 and configure bigger than 5 to avoid throttling
```

Comment:
Let's make this 10 seconds by default as GCP Monitoring doesn't want more time series anyway. (https://cloud.google.com/monitoring/quotas)

Reply:
Sure. no problem and thanks for the detailed review.